### DISCUSSION TITLE
Form.io js in directus.

### DISCUSSION THREAD

**Discussion by phxgbl** [directus/directus#6523] [Feature Requests]
I am new to directus and am impressed by the features. I have a use case in which forms should be dynamic and even driven ( eg: onchange event on a select should change the values of other fields etc). Directus forms are definitely dynamic but may have to improve in their event-driven capability.  I have hands-on experience in Form.io(https://github.com/formio/formio.js) which is flexible and MIT licensed. It is possible to use formio form in an external application and connect to directus using API calls but my idea or question here is, is there an option to use formio form in the core. I am ready to do changes to the code. If it is possible, can you guide or give insight on how to implement the same in directus?

**Comment by benhaynes** [comment]
This functionality will be included with the addition of Conditional Fields, which is tracked here: https://github.com/directus/directus/issues/3151

Since this ticket is sponsored, it will be released in the coming month or so.

**Comment by rijkvanzanten** [comment]
> is there an option to use formio form in the core.

You could make a custom module using form.io if you so desire, though like @benhaynes mentioned, #3151 should take care of your use case. 

We have no intention to switch to form.io for Directus' main form grid at this time 👍🏻

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
